docker/etc/nginx/custom/headers.conf

```diff
@@ -1,5 +1,5 @@
 # Add security headers.
-add_header Content-Security-Policy "worker-src 'self' blob:; script-src 'unsafe-inline'; img-src 'self' data: https:; font-src https: fonts.googleapis.com; report-uri /report-csp-violation; upgrade-insecure-requests";
+add_header Content-Security-Policy "worker-src 'self' blob:; script-src 'self' 'unsafe-inline'; img-src 'self' data: https:; font-src https: fonts.googleapis.com; report-uri /report-csp-violation; upgrade-insecure-requests";
```

Comment:
AFAIK we don't have a reporting endpoint?

Reply:
I don't know how useful the reporting endpoint is - we could monitor page not found logs for `/report-csp-violation` or we can remove `report-uri /report-csp-violation; ` from the directive completely.

We _could_ set up a page to handle those reports, https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP#enabling_reporting but I don't think it gives much more information than the console log.